kernel/core/os.adb

```diff
@@ -481,8 +481,9 @@ is
                 Input => (os_task_ro, os_task_current)),
       Pre => os_ghost_task_list_is_well_formed and
              os_ghost_current_task_is_ready,
+             os_ghost_task_mbx_are_well_formed (dest_id),
```

Comment:
oops copy paste error, should have an additional "and" operator:
              os_ghost_current_task_is_ready and
+             os_ghost_task_mbx_are_well_formed (dest_id),
As os_ghost_task_mbx_are_well_formed is just for one task, shouldn't be easier for the prover to match it directly rather than inside os_ghost_mbx_are_well_formed which is inside os_ghost_task_list_is_well_formed?
I don't really know, should not be. Something to confirm with Adacore.